ibis/backends/dask/execution/window.py

```diff
@@ -84,3 +99,67 @@ def execute_window_op(
         **kwargs,
     )
     return _post_process_empty(result, data, timecontext)
+
+
+def execute_grouped_window_op(
+    op, data, window, scope, timecontext, aggcontext, clients, **kwargs,
+):
+    # extract the parent
+    (root,) = op.root_tables()
+    root_expr = root.to_expr()
+
+    data = execute(
```

Comment:
Why re-executing data?

Reply:
`data` actually becomes the parent df, because we're executing on the root_expr (basically the logic in https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/execution/window.py#L275-L285). 

Before execute:
```
>>> data
Dask Series Structure:
npartitions=2
0             float64
4294967296        ...
8589934592        ...
Name: double_col, dtype: float64
```
after execute
```
>>> data
Dask DataFrame Structure:
               index Unnamed: 0     id bool_col tinyint_col smallint_col int_col bigint_col float_col double_col date_string_col string_col   timestamp_col   year  month
npartitions=2                                                                                                                                                            
0              int64      int64  int64     bool       int64        int64   int64      int64   float64    float64          object     object  datetime64[ns]  int64  int64
4294967296       ...        ...    ...      ...         ...          ...     ...        ...       ...        ...             ...        ...             ...    ...    ...
8589934592       ...        ...    ...      ...         ...          ...     ...        ...       ...        ...             ...        ...             ...    ...    ...
```

Reply:
Can we not override `data`? It seems a bit confusing just to reuse that variable. If you need to execute the root table let's give it another variable name